apiary.apib

```diff
@@ -5507,3 +5507,136 @@ and [buckets](https://keboola.docs.apiary.io/#reference/buckets/manage-bucket/bu
         {
             "displayName": "My-Display-Name"
         }
+
+# Group Direct Access
+**Direct Access** allows you to connect directly to the your project backend(for now only snowflake is supported).
+Then you are allowed to works with your tables(call query, create new tables etc.) in buckets which has turned on **Direct Access**.
+These tables are automatically updated when something change in source buckets(tables) in Keboola Connection.
+Each user in project can have its own credentials for **Direct Access**. The credentials are valid per project,
+if you want **Direct Access** to the another project, you have to create it by `StorageApi-Token`(Only admin token is allowed) for project where
+you want to have **Direct Access**.
+
+## Create or List Direct Access Credentials [/v2/storage/direct-access/{backend}]
+
++ Parameters
+    + backend (required, enum[string]) - type of backend
+        + Members
+            + snowflake
+
+###Create Credentials [POST]
+Creates and new credentials(for all supported backends) in your project to **Direct Access** connect for buckets.
+
++ Request
+
+    + Headers
+
+            X-StorageApi-Token: your_token
+
++ Response 201 (application/json)
+
+        {
+            "snowflake": {
+                "host": "keboola.snowflake.com"
+                "username": "KEBOOLA_DA_1_1",
+                "password": "q.s3JQLTUC8i2v_BQQC3jLChqmz9L2ic"
```

Comment:
To heslo je změněné, žejo...

Reply:
ja som si len vygeneroval v 1P 32 znakove heslo aby bolo jasne ze sa tam dava take dlhe heslo.